mage_integrations/mage_integrations/connections/utils/google.py

```diff
@@ -1,4 +1,4 @@
-from typing import TypedDict
+from typing_extensions import TypedDict
```

Comment:
@wangxiaoyou1993 Python 3.7 had an issue with the `typing` module so needed to use the `typing_extensions` module instead to resolve that error. Is this fine?

Reply:
We use typing in a lot of places and it didn't have issues. What's the issue here?

Reply:
@wangxiaoyou1993  This error came up when running the tests for Python 3.7 specifically. Should work okay for 3.8 and 3.9. https://github.com/mage-ai/mage-ai/actions/runs/4056957060/jobs/6982109766

Reply:
ok, got it. then you can use typing_extensions here.